robot_smach_states/examples/human_interaction/example_people_recognition.py

```diff
@@ -0,0 +1,85 @@
+# System
+import argparse
+
+# ROS
+import rospy
+import smach
+
+# TU/e Robotics
+from robot_skills.get_robot import get_robot
+from robot_skills.robot import Robot
+
+# Robot Smach States
+from robot_smach_states.util.designators.core import VariableDesignator
+from robot_smach_states.human_interaction import LearnPerson, RecognizePerson, Say
+
+
+class ExamplePeopleRecognition(smach.StateMachine):
+    def __init__(self, robot: Robot):
+        """
+        Example statemachine to test people learning and recognition
+
+        :param robot: robot API object
+        """
+        smach.StateMachine.__init__(self, outcomes=["Done"])
+
+        name_designator = VariableDesignator(resolve_type=str)
+
+        with self:
+            smach.StateMachine.add(
```

Comment:
It might be lipservice but I think it might be handy for the robot to say "I will now learn." since the operator will be away from the laptop they might not be able to monitor the terminal

Reply:
This was already present in the `LearnPerson` state (see [human_interaction.py](https://github.com/tue-robotics/tue_robocup/blob/master/robot_smach_states/src/robot_smach_states/human_interaction/human_interaction.py). Therefore I considered it overkill